File: src/python3/001. Two Sum/TwoSum.py
# 1. Two Sum
# Accepted 4228ms

class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i, vi in enumerate(nums):
            for j, vj in enumerate(nums[i + 1:]):
                if vi + vj == target:
                    ret = [i, j + i + 1]
                    break
            else:
                continue
            break
        else:
            ret = [0, 0]
        
        return ret
        

File: src/javascript/001. Two Sum/twoSum.js
// 1. Two Sum
// Accepted 108ms

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let leftIdx = 0;
    let rightIdx = 0;
    
    outer_loop:
    for(let k = 0; k < nums.length; ++k) {
        for(let j = k + 1; j < nums.length; ++j) {
            if(nums[k] + nums[j] == target) {
                leftIdx = k;
                rightIdx = j;
                break outer_loop;
            }
        }
    }
    
    return [leftIdx, rightIdx];
};
